### DISCUSSION TITLE
I lost my .nuxt folder after updating...

### DISCUSSION THREAD

**Discussion by Jopgood** [nuxt/nuxt#25157] [Questions]
Hi there!

I am a little new to Nuxt but do have some experience. I am using NPM for package management and have a working web app. I wanted to install the package 'nuxt-directus-next' for managing my Directus instance and found that I was getting override errors regarding my Vue version?

```
npm ERR! code EOVERRIDE
npm ERR! Override for vue@^3.4.3 conflicts with direct dependency
```

I tried doing npx nuxi upgrade and npx nuxi upgrade --force but would continue to get the same error. I removed the overrides object in my package.json and now npx nuxi upgrade was working! However, I went to boot up my webapp and found that it didn't load and there were many errors. Upon looking in the nuxt.config.ts (and other places) I found lots of autoimports not working and that's when I noticed the .nuxt folder was missing which I believe had all the autoimports stored as well as the majority of the tsconfig.json data.

It's evident I've messed it up myself somewhere here but not sure what I have done wrong and how to fix it!

Any help/advice would be greatly appreciated and welcomed as I am still learning!

**Comment by angelhdzmultimedia** [comment]
.nuxt directory is auto generated every time you execute `npm dev`. In some cases you might need to delete it, and then run nuxt to generate it back with the correct types.
